Create liveramp-cohort.md (#36)
amplitude/amplitude-dev-center: docs/data/destinations/liveramp-cohort.md

---
title: Send Cohorts to Liveramp
description: Sync cohorts from Amplitude to Liveramp
---

--8<-- "includes/editions-all-paid-editions.md"

[LiveRamp](https://liveramp.com/) is a data connectivity platform that enables businesses to connect, unify, control, and activate data across different channels and devices to optimize customer experiences and drive better results.

> This guide is your resource for sending cohorts from Amplitude to LiveRamp, a powerful data connectivity platform. By following the instructions outlined here, you can seamlessly synchronize and activate your cohorts within LiveRamp. This process enables you to leverage LiveRamp's extensive network of integrations with top measurement and media partners, spanning over 550+ destinations. Whether you opt for sending cohorts via Secure File Transfer Protocol (SFTP), Amazon S3, or Snowflake, this guide provides a step-by-step walkthrough to enhance your customer experiences and achieve superior results while reaching a vast array of platforms and services.

Author: Updated it and made it more brief :)

## Considerations

- Amplitude lacks a built-in LiveRamp destination.
- A paid LiveRamp license is required for this integration.

> ## Setup

Reviewer: ```suggestion
```
I don't think we need this heading here.

> ### Send cohort via Secure File Transfer Protocol (SFTP) 

Reviewer: ```suggestion
## Send a cohort through Secure File Transfer Protocol (SFTP) 

To send a cohort from Amplitude to LiveRamp through SFTP:
```

> 1. **Create Cohort in Amplitude:** Start by creating the cohort you want to sync in Amplitude.

Reviewer: ```suggestion
1. **Create the Cohort in Amplitude:** If the cohort you want to send does not exist, create it in Amplitude.
```

> 2. **Export Cohort from Amplitude:** Export the cohort as a CSV file or via the Behavioral Cohorts API. See this [document](https://help.amplitude.com/hc/en-us/articles/360028552471-Amplitude-Audiences-overview-Drive-conversions-with-true-one-to-one-personalization-) for more details.

Reviewer: ```suggestion
2. **Export the Cohort from Amplitude:** Export the cohort as a CSV file or with the Behavioral Cohorts API. See this [document](https://help.amplitude.com/hc/en-us/articles/360028552471-Amplitude-Audiences-overview-Drive-conversions-with-true-one-to-one-personalization-) for more details.
```

What details should users look for in the linked article? We should point directly to content about CSV export or the Behavioral Cohorts API.

Also, please use the article title as the link text, instead of `document`. For example:

`For more information, see [Article title | Article section](url)`

> 3. **Upload Cohort to SFTP Server:** Securely upload the exported cohort file to your SFTP server. Refer to your [SFTP server documentation](https://docs.liveramp.com/connect/en/upload-a-file-via-liveramp-s-sftp.html) for guidance around the file requirements.

Reviewer: ```suggestion
3. **Upload the Cohort to SFTP Server:** Upload the exported cohort file to LiveRamp's SFTP server. See LiveRamp's article [Upload a File via LiveRamp's SFTP](https://docs.liveramp.com/connect/en/upload-a-file-via-liveramp-s-sftp.html) for more information. LiveRamp may take up to 3 days to ingest your cohort data from the SFTP location.
```

>     - Liveramp can take up to 1-3 days to ingest the data from SFTP into Liveramp.

Reviewer: ```suggestion
```

> 4. **Choose Destination Account:** In LiveRamp, select a destination account based on your media plan (e.g., Facebook Ads) for cohort activation.

Reviewer: Is there LiveRamp documentation we can point to for these last two steps? Documenting LiveRamp features in our docs is not ideal.

Author: Agreed! I forgot to add the link. I originally had a google doc where I created the draft.

5. **Add Cohort to Destination:** Add the uploaded cohort to the chosen destination account in LiveRamp for activation.

> ### Send cohort via Amazon S3

Reviewer: ```suggestion
### Send a cohort with Amazon S3

To send a cohort from Amplitude to LiveRamp through an Amazon S3 bucket:
```

> 1. **Create Cohort in Amplitude:** Create the desired cohort within Amplitude.

Reviewer: ```suggestion

1. **Create the Cohort in Amplitude:** If the cohort you want to send does not exist, create it in Amplitude.
```

> 2. **Set Up Amazon S3 Bucket:** Configure your Amazon S3 bucket if you don't have one already, using the AWS Management Console.

Reviewer: ```suggestion
2. **Configure the Amazon S3 Bucket:** Use the AWS Management Console to create an S3 bucket, if you don't have one.
```

> 3. **Send Cohort to Amazon S3:** Use the [Amazon S3 cohort sync](https://www.docs.developers.amplitude.com/data/destinations/amazon-s3-cohort/) integration to sync cohort data from Amplitude to your Amazon S3 bucket.

Reviewer: ```suggestion
3. **Send the Cohort to Amazon S3:** Use the [Amazon S3 cohort sync](https://www.docs.developers.amplitude.com/data/destinations/amazon-s3-cohort/) integration to sync cohort data from Amplitude to your Amazon S3 bucket.
```

> 4. **Log Into LiveRamp:** Access your LiveRamp account.

Reviewer: Does LiveRamp have documentation about how to retrieve the cohort data from S3?

Author: Updated the link.

5. **Find the Cohort:** Locate the cohort within your LiveRamp account.
6. **Choose Destination Account:** Select a destination account aligned with your media plan (e.g., Facebook Ads) for cohort activation.
7. **Add Cohort to Destination:** Add the cohort to the chosen destination account within LiveRamp to activate it.

> ## Send cohort via Snowflake

Reviewer: ```suggestion
## Send a cohort with Snowflake

To send a cohort from Amplitude to LiveRamp with Snowflake:
```

> 1. **Send Cohorts to Amazon S3:** Send cohorts from Amplitude to Amazon S3. See this document for more details.

Reviewer: What document?

Author: Updated the link.

> 2. **Bulk Loading from Amazon S3:** Load cohort data from your Amazon S3 bucket into your Snowflake data warehouse. See this document for more details.

Reviewer: ```suggestion
2. **Bulk Load from Amazon S3:** Load cohort data from your Amazon S3 bucket into your Snowflake data warehouse. See this document for more details.
```

What document?

> 3. **Install LiveRamp Native Application on Snowflake:** Install the LiveRamp native application in your Snowflake environment.

Reviewer: Again, we should link to LiveRamp documentation here, so we are responsible for updates if instructions change on their side.

Author: Updated the link.

---
title: Send Cohorts to Liveramp
description: Sync cohorts from Amplitude to Liveramp
---

--8<-- "includes/editions-all-paid-editions.md"

[LiveRamp](https://liveramp.com/) is a data connectivity platform that enables businesses to connect, unify, control, and activate data across different channels and devices to optimize customer experiences and drive better results.

This guide explains how to integrate Amplitude with LiveRamp, offering three methods: SFTP, Amazon S3, and Snowflake. These methods facilitate the transfer and activation of cohorts within LiveRamp, providing flexibility in your data connectivity options.

## Considerations

> - A paid LiveRamp license is required for this integration.

Reviewer: ```suggestion
A paid LiveRamp license is required for this integration.
```

## Send a cohort through Secure File Transfer Protocol (SFTP) 

To send a cohort from Amplitude to LiveRamp through SFTP:

1. **Create the Cohort in Amplitude:** If the cohort you want to send does not exist, create it in Amplitude.
2. **Export the Cohort from Amplitude:** Export the cohort as a CSV file or with the Behavioral Cohorts API. See this [document](https://help.amplitude.com/hc/en-us/articles/360028552471-Amplitude-Audiences-overview-Drive-conversions-with-true-one-to-one-personalization-) for more details.
3. **Upload the Cohort to SFTP Server:** Upload the exported cohort file to LiveRamp's SFTP server. See LiveRamp's article [Upload a File via LiveRamp's SFTP](https://docs.liveramp.com/connect/en/upload-a-file-via-liveramp-s-sftp.html) for more information. LiveRamp may take up to 3 days to ingest your cohort data from the SFTP location.

> 4. **Choose Destination Account:** In LiveRamp, select a destination account based on your media plan (e.g., Facebook Ads) for cohort activation.

Reviewer: ```suggestion
4. **Choose Destination Account:** In LiveRamp, select a destination account based on your media plan (for example, Facebook Ads) for cohort activation.
```

5. **Add Cohort to Destination:** Add the cohort to the chosen destination account within LiveRamp to activate it.

### Send a cohort with Amazon S3

To send a cohort from Amplitude to LiveRamp through an Amazon S3 bucket:

> 1. **Create the Cohort in Amplitude:** If the cohort you want to send does not exist, create it in Amplitude.

Reviewer: ```suggestion
1. **Create the Cohort in Amplitude:** If the cohort you want to send doesn't exist, create it in Amplitude.
```

2. **Configure the Amazon S3 Bucket:** Use the AWS Management Console to create an S3 bucket, if you don't have one.
3. **Send the Cohort to Amazon S3:** Use the [Amazon S3 cohort sync](https://www.docs.developers.amplitude.com/data/destinations/amazon-s3-cohort/) integration to sync cohort data from Amplitude to your Amazon S3 bucket.

> 4. **Grant Liveramp access to your Amazon S3 bucket:** Allow LiveRamp to Access Your AWS S3 Bucket. See this [document](https://docs.liveramp.com/connect/en/allow-liveramp-to-access-your-aws-s3-bucket.html) here.

Reviewer: ```suggestion
4. **Grant LiveRamp access to your Amazon S3 bucket:** Allow LiveRamp to Access Your AWS S3 Bucket. For more information, see LiveRamp's article, [Allow LiveRamp to Access your AWS S3 Bucket](https://docs.liveramp.com/connect/en/allow-liveramp-to-access-your-aws-s3-bucket.html)
```

5. **Log Into LiveRamp:** Access your LiveRamp account.
6. **Find the Cohort:** Locate the cohort within your LiveRamp account.

> 7. **Choose Destination Account:** In LiveRamp, select a destination account based on your media plan (e.g., Facebook Ads) for cohort activation.

Reviewer: ```suggestion
7. **Choose Destination Account:** In LiveRamp, select a destination account based on your media plan (for example, Facebook Ads) for cohort activation.
```

> 8. **Add Cohort to Destination:** Add the cohort to the chosen destination account within LiveRamp to activate it.

Reviewer: ```suggestion
8. **Add Cohort to Destination:** Add the cohort to the chosen destination account in LiveRamp to activate it.
```

## Send a cohort with Snowflake

To send a cohort from Amplitude to LiveRamp with Snowflake:

> 1. **Send Cohorts to Amazon S3:** Send cohorts from Amplitude to Amazon S3. See this [document](https://www.docs.developers.amplitude.com/data/destinations/amazon-s3-cohort/) for more details.

Reviewer: ```suggestion
1. **Send Cohorts to Amazon S3:** Send cohorts from Amplitude to Amazon S3. For more information, see [Send Cohorts to Amazon S3](https://www.docs.developers.amplitude.com/data/destinations/amazon-s3-cohort/)
```

> 2. **Bulk Load from Amazon S3:** Load cohort data from your Amazon S3 bucket into your Snowflake data warehouse. See this [document](https://docs.snowflake.com/en/user-guide/data-load-s3) for more details.

Reviewer: ```suggestion
2. **Bulk Load from Amazon S3:** Load cohort data from your Amazon S3 bucket into your Snowflake data warehouse. For more information, see Snowflake's article, [Bulk Loading from Amazon S3](https://docs.snowflake.com/en/user-guide/data-load-s3).
```

> 3. **Install LiveRamp Native Application on Snowflake:** Install the LiveRamp native application in your Snowflake environment. See this [document](https://docs.liveramp.com/identity/en/set-up-the-liveramp-native-app-in-snowflake.html) for more details.

Reviewer: ```suggestion
3. **Install LiveRamp Native Application on Snowflake:** Install the LiveRamp native application in your Snowflake environment. For more information, see LiveRamp's article, [Set Up the LiveRamp Native App in Snowflake](https://docs.liveramp.com/identity/en/set-up-the-liveramp-native-app-in-snowflake.html).
```

> 4. **Enable LiveRamp Embedded Identity in Snowflake:** Activate LiveRamp's Identity Resolution in Snowflake to translate identifiers to RampIDs. See this [document](https://docs.liveramp.com/identity/en/perform-identity-resolution-in-snowflake.html) for more details.

Reviewer: ```suggestion
4. **Enable LiveRamp Embedded Identity in Snowflake:** Activate LiveRamp's Identity Resolution in Snowflake to translate identifiers to RampIDs. For more information, see LiveRamp's article [Perform Identity Resolution in Snowflake](https://docs.liveramp.com/identity/en/perform-identity-resolution-in-snowflake.html).
```

> 5. **Share Data to LiveRamp Account:** Share the processed data with your LiveRamp account.

Reviewer: How should a user do this? This step doesn't provide any instruction as written.

> 6. **Choose Destination Account:** In LiveRamp, select a destination account based on your media plan (e.g., Facebook Ads) for cohort activation.

Reviewer: ```suggestion
6. **Choose Destination Account:** In LiveRamp, select a destination account based on your media plan (for example, Facebook Ads) for cohort activation.
```
